packages/client/hmi-client/src/workflow/tera-workflow.vue

```diff
@@ -583,6 +584,72 @@ function onPortMouseleave() {
 	isMouseOverPort = false;
 }
 
+function resizeHandler(node: WorkflowNode<any>) {
+	relinkEdges(node);
+}
+
+// For relinking
+const dist2 = (a: Position, b: Position) => (a.x - b.x) * (a.x - b.x) + (a.y - b.y) * (a.y - b.y);
+const threshold2 = 5.0 * 5.0;
+
+/**
+ * Relink edges that have become detatched
+ *
+ * [output-port](edge source => edge target)[input-port]
+ *
+ * FIXME: not efficient, need cache/map for larger workflows
+ * */
```

Comment:
```suggestion
 **/
```

Reply:
I think this is prettier being annoying and decide to format comments.